dev-infrastructure/configurations/mvp-svc-cluster.bicepparam

```diff
@@ -32,6 +32,9 @@ param acrPullResourceGroups = ['global']
 param clustersServiceAcrResourceGroupNames = ['global']
 param imageSyncAcrResourceGroupNames = ['global']
 
+param oidcStorageAccountName = 'arohcpoidcdev'
+param aroDevopsMsiId = '/subscriptions/1d3378d3-5a3f-4712-85a1-2485495dfc4b/resourceGroups/global/providers/Microsoft.ManagedIdentity/userAssignedIdentities/aro-hcp-devops'
```

Comment:
Aren't we usually doing a lookup on these kind of values using resource `existing`?

Reply:
i noticed that MSFT is providing this kind of MSI refs as resource ID, not just as name. this way they are flexible to have this MSI in whatever subscription/RG.

i would like to adapt our other code that uses such things to this pattern as well, e.g. the certofficer MSI for key vaults or the MSI that manages PG entra access.